app/views/govuk_publishing_components/components/docs/autocomplete.yml

```diff
@@ -0,0 +1,78 @@
+
+name: Autocomplete (experimental)
+description: The autocomplete component lets users select from a dynamic list of suggestions to populate the value of a search input element.
+body: |
+  This component does not perform any matching of user input to refine suggestions. The assumption is that the backend that the query string is supplied to does the matching and returns only results that are appropriate.
```

Comment:
Nitpick: sentence structure here is a bit confusing... is it trying to say `The assumption is that the backend returns only results that match the query string.`?